movie_api/settings.py

```diff
@@ -0,0 +1,108 @@
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-t-r6-=-^39(b_z)$ax^4qox_dmne4pq6j#@9dg&xgrmm7gtaqp"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Security Issue: `DEBUG` is set to `True`. Ensure this is set to `False` in a production environment to avoid exposing sensitive information.